testing/integration/src/builtin.rs

```diff
@@ -0,0 +1,103 @@
+use std::collections::BTreeMap;
+
+use anyhow::{Context, Result};
+use cid::Cid;
+use futures::executor::block_on;
+use fvm::state_tree::{ActorState, StateTree};
+use fvm::{init_actor, system_actor};
+use fvm_ipld_blockstore::{Blockstore, MemoryBlockstore};
+use fvm_ipld_car::load_car;
+use fvm_ipld_encoding::CborStore;
+use fvm_shared::actor::builtin::{load_manifest, Type};
+use fvm_shared::version::NetworkVersion;
+use multihash::Code;
+
+use crate::error::Error::{
+    FailedToLoadManifest, FailedToSetActor, FailedToSetState, MultipleRootCid, NoCidInManifest,
+};
+
+const BUNDLES: [(NetworkVersion, &[u8]); 2] = [
+    (NetworkVersion::V14, actors_v6::BUNDLE_CAR),
+    (NetworkVersion::V15, actors_v7::BUNDLE_CAR),
+];
+
+// Import built-in actors
+pub fn import_builtin_actors(
```

Comment:
The reason was mostly because I saw it as utilities more than proper method that we wanted to access as a tester. If we were to add actors bundle as parameters for `Tester` it would make sense but I did not see that case happened.

Reply:
Understood.